src/app/view/components/_common/push-publish-dialog/push-publish-dialog.component.html

```diff
@@ -0,0 +1,48 @@
+<p-dialog
+    modal="modal"
+    width="500"
+    [dismissableMask]="true"
+    [(visible)]="displayDialog"
+    [responsive]="true"
+    [resizable]="false"
+    [draggable]="false"> 
+
+    <p-header>
+        {{dotMessageService.get('contenttypes.content.push_publish')}}
+    </p-header>
+
+    <form [formGroup]="form" (ngSubmit)="submitPushAction($event)" novalidate>
+        <div class="form__group">
+            <label class="form__label">{{dotMessageService.get('contenttypes.content.push_publish.I_want_To')}}: </label>
+            <p-dropdown [style]="{'width':'100%'}" [options]="pushActions" (onChange)="pushActionChange($event)" formControlName="pushActionSelected"></p-dropdown>
+        </div>
+
+        <div class="form__group" *ngIf="publish">
+            <input type="checkbox" [(ngModel)]="forcePushCheckbox" formControlName="forcePush">
+            <label>{{dotMessageService.get('contenttypes.content.push_publish.force_push')}}</label>
+        </div>
+
+        <div class="form__group push-publish-dialog__publish-date" *ngIf="publish">
+            <label class="form__label">{{dotMessageService.get('contenttypes.content.push_publish.publish_date')}}: </label>
+            <p-calendar [(ngModel)]="publishdate" [showTime]="true" [style]="{'width':'100%'}" formControlName="publishdate"></p-calendar>
+        </div>
+
+        <div class="form__group push-publish-dialog__expire-date" *ngIf="showExpireDate">
+            <label class="form__label">{{dotMessageService.get('contenttypes.content.push_publish.expire_date')}}: </label>
+            <p-calendar [(ngModel)]="expiredate" [showTime]="true" [style]="{'width':'100%'}" formControlName="expiredate"></p-calendar>
+        </div>
+
+        <div class="form__group">
+            <label class="form__label">{{dotMessageService.get('contenttypes.content.push_publish.push_to')}}: </label>
+            <dot-push-publish-env-selector [contentTypeId]="contentTypeId" formControlName="environment"></dot-push-publish-env-selector>
+            <small [hidden]="form.controls.environment.valid || (form.controls.environment.pristine && !submitted)" class="push-publish-dialog__error-message">
+                {{dotMessageService.get('contenttypes.content.push_publish.push_to_errormsg')}}
+            </small>
+        </div>
+
+        <p-footer class="push-publish-dialog__footer-form">
+            <button pButton type="button" flat (click)="close($event)" label="{{dotMessageService.get('contenttypes.content.push_publish.form.cancel')}}" class="push-publish-dialog__form-cancel"></button>
```

Comment:
We don't need flat here, I made a chance to make all the buttons in a dialog flat by default.